### PR TITLE
:arrow_up: Update @process-engine/skeleton-electron to 2.0.0

### PR DESCRIPTION
**Pull request by Paulomart**
## What did you change?

- Upgrades the @process-engine/skeleton-electron dependency to 2.0.0
- Catch any errors during startup and display notification

⏰ Depends on https://github.com/process-engine/skeleton/pull/46

## How can others test the changes?

Build and see that the `Import process` function is working.

See https://github.com/process-engine/bpmn-studio/pull/294

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

**Comment by Paulomart** [comment]
Dont merge this PR yet, the version `1.1.1` of `@process-engine/skeleton-electron` was unpublished.

**Comment by Paulomart** [comment]
We need to bump the version of `@process-engine/skeleton-electron` to 2.0.0, because it contains a breaking change. Because of the breaking change in it, bpmn-studio will need a major version bump as well.

File: src/main.ts [hunk 1]
```diff
@@ -52,5 +54,23 @@ export function configure(aurelia: Aurelia): void {
...
+      // subscribe to processengine status
+      ipcRenderer.send('add_internal_processengine_status_listener');
+      // wait for status to be reported
+      ipcRenderer.on('internal_processengine_status', (event: any, status: string, error: string) => {
+        if (status !== 'error') {
+          return;
+        }
+        const errorMessage: string = `Failed to start ProcessEngine.`;
+        const notificationService: NotificationService = aurelia.container.get('NotificationService');
+        notificationService.showNotification(NotificationType.ERROR, errorMessage);
```
Comment (alexanderkasten) [inline src/main.ts]: Why you only show `Failed to start ProcessEngine`? You could show the error which occurs, that helps the user to understand why the start fails.

File: src/main.ts [hunk 2]
```diff
@@ -52,5 +54,23 @@ export function configure(aurelia: Aurelia): void {
...
+    if ((<any> window).nodeRequire) {
+      const ipcRenderer: any = (<any> window).nodeRequire('electron').ipcRenderer;
+      // subscribe to processengine status
+      ipcRenderer.send('add_internal_processengine_status_listener');
+      // wait for status to be reported
+      ipcRenderer.on('internal_processengine_status', (event: any, status: string, error: string) => {
+        if (status !== 'error') {
+          return;
+        }
+        const errorMessage: string = `Failed to start ProcessEngine.`;
```
Comment (SteffenKn) [inline src/main.ts]: Do not use back-ticks for this. Better use single quotes like that:
'Failed to start ProcessEngine.'

File: electron_app/electron.js [hunk 3]
```diff
@@ -19,7 +20,75 @@ if (!isDev) {
...
+    if (internalProcessEngineStatus !== undefined) {
+      sendInternalProcessEngineStatus(event.sender);
+    }
+  });
+
+  // TODO: Check if the ProcessEngine instance is now run on the UI thread.
+  // See issue https://github.com/process-engine/bpmn-studio/issues/312
+  startProcessEngine()
+    .then((processengine) => {
+      console.log(`Internal ProcessEngine started successfully.`);
```
Comment (SteffenKn) [inline electron_app/electron.js]: Do not use back-ticks for this. Better use single quotes like that:
'Internal ProcessEngine started successfully.'

**Pull request merged**